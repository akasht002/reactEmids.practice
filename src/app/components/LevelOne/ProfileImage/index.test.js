import React from 'react';
import {ProfileImage} from './index';
import renderer from 'react-test-renderer';

test('ProfileImage Component testing', () => {
    const component = renderer.create(
      <ProfileImage></ProfileImage>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });