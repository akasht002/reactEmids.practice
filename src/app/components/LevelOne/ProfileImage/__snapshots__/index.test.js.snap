// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileImage Component testing 1`] = `
<div
  className="ProfileImageWidget"
>
  <div
    className="ProfileImageContainer"
  >
    <svg
      className="circular-chart"
      viewBox="1.55 1.55 33 33"
    >
      <path
        className="circle"
        d="M18 2.0845 a 15.9155 15.9155 0 0 1 0 31.831 a 15.9155 15.9155 0 0 1 0 -31.831"
        strokeDasharray="undefined,100"
      />
    </svg>
    <img
      alt="user"
      className="ProfileImage"
    />
  </div>
</div>
`;
